examples/ssh/linux_services.js

```diff
@@ -0,0 +1,133 @@
+/** Name: Linux Services Monitoring
+ * Description: Monitors the status of all the services on a Linux machine
+ *   
+ * Communication protocol is SSH, using the Linux bash command.
+ * 
+ * Tested on Linux with the following specs:
+ *      - Debian Linux 10
+ *      - bash version 5.0.3(1)
+ * 
+ * Requires:
+ *      - systemctl
+ *      - sed, grep, awk, and tail
+ *      - PLEASE NOTE: it requires to be rut as root
+ * 
+ * 
+ * Creates a Custom Driver Table with the following columns:
+ *  - Service Name
+ *  - Service Status
+ * 
+**/
+
+// Define the services you want to show (all|include)
+var servicesToGet="include"; 
```

Comment:
This is not an intuitive way of defining an API to include/exclude services to be monitored,

Please follow the same approach as windows_services_status.js where we use List/Array variable instead of a textual one, which is error prone for lists